material_3_demo/pubspec.yaml

```diff
@@ -9,7 +9,6 @@ version: 1.0.0+1
 
 environment:
   sdk: ^3.5.0
-  flutter: ^3.19.0
```

Comment:
We shouldn't pin versions in the samples repo

Reply:
That's not a pin, but a minimum version. I'm guessing this sample relies on something that was introduced in that Flutter version.

Reply:
You're right, I guess I wasn't paying attention. One way or the other, the analyzer wasn't highlighting the warnings that I know were there. Removing this probably wasn't what fixed it, I'll revert